utils/scapp.h

```diff
@@ -630,18 +641,26 @@ class scApp : public ktxApp {
           "               Disable RDO multithreading (slightly higher compression,\n"
           "               deterministic).\n\n"
           "  --normal_mode\n"
-          "               For ASTC encoder '--encode astc' assumes the input texture is\n"
-          "               a three component linear LDR normal map storing unit length\n"
-          "               normals as (R=X, G=Y, B=Z). The output will be a two component\n"
-          "               X+Y normal map stored as (RGB=X, A=Y), optimized for angular\n"
-          "               error instead of simple PSNR. The Z component can be recovered\n"
-          "               programmatically in shader code by using the equation:\n\n"
+          "               Only valid for linear textures with two or three components.\n"
+          "               If the input texture has three linear components it is assumed to\n"
+          "               be a three component linear normal map storing unit length\n"
+          "               normals as (R=X, G=Y, B=Z). It will be converted to a two \n"
+          "               component X+Y normal map stored as (RGB=X, A=Y) prior to\n"
+          "               encoding.  If the input has 2 linear components it is assumed to\n"
```

Comment:
s/. If/. If unsure that your normals are unit length, use\n"
"                    '--normalize'. If the input has 2 linear components it is assumed to\n"/

Reply:
fixed in https://github.com/KhronosGroup/KTX-Software/pull/493/commits/d92aa03a96ee6fa6b858d8e7315e1b0a61f8e5fb